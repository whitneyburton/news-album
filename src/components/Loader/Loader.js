import React from 'react';

export const Loader = () => {
  return (
    <div className="Loader">
      <h2>Loading...</h2>
    </div>
  );
};
