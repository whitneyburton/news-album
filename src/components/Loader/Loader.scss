.Loader {
  display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center;
  margin-top: 15%;
  font-size: 35px;
}
