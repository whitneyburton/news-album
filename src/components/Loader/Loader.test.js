import React from 'react';
import { shallow } from 'enzyme';
import { Loader } from './Loader';

describe('Loader', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
