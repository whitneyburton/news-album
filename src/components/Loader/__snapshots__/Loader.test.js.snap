// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `
<div
  className="Loader"
>
  <h2>
    Loading...
  </h2>
</div>
`;
